frontend/src/pages/AdminPage/tableColumns.tsx

```diff
@@ -28,8 +28,16 @@ const createColumns: CreateColumns = (
     accessor: "aac"
   },
   {
-    Header: "User",
-    accessor: "username"
+    Header: "First Name",
+    accessor: "firstName"
+  },
+  {
+    Header: "Last Name",
+    accessor: "lastName"
+  },
+  {
+    Header: "DODID",
+    accessor: "dodID"
```

Comment:
Based on #315, these will likely end up coming from the api as:

`username` -> dod id
`first_name` -> users first name
`last_name` -> users last name

You could either restructure it in the processing step in `useDocumentApi.ts`, or update these accessors accordingly

Reply:
I'll make the change throughout the site for username to be dod id